### PR TITLE
Block phishing IPs

### PR DESCRIPTION
**Pull request by Yuki2718**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report
- [x] My code follows [syntax](https://adguard-dns.io/kb/general/dns-filtering-syntax/) of this project
- [x] I have performed a self-review of my own changes
- [x] My changes do not break web sites, apps and files structure

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads
- [ ] Website or app doesn't work properly
- [ ] Missed analytics or tracker
- [ ] Filters maintenance

## What issue is being fixed?

https://github.com/AdguardTeam/AdguardFilters/pull/160415/
https://github.com/AdguardTeam/AdguardFilters/pull/160610/

### Add your comment and screenshots

1. Your comment

Block them by IPs

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: Filters/rules.txt [hunk 1]
```diff
@@ -573,6 +573,12 @@
 !
 ! Special block section, requires explanation for the rule
 !
+! https://github.com/AdguardTeam/AdguardFilters/pull/160610
+199.167.138.162
```
Comment (Alex-302) [inline Filters/rules.txt:577]: As I remember, by DNS filtering can be blocked only domain, not IPs.

Comment (Yuki2718) [reply Filters/rules.txt:577]: You saw Slack discussion, right? At least working on adguard DNS. Will check on apps.

Comment (Yuki2718) [reply Filters/rules.txt:577]: Working on AG for Android too.

Comment (Yuki2718) [reply Filters/rules.txt:577]: Working, e.g. latest tweet `https://twitter.com/NaomiSuzuki_/status/1699788108894728501`. They're revolving. And twitter search `199.167.138.162` `199.167.138.163` indicates the IP has been stably used.

Comment (Alex-302) [reply Filters/rules.txt:577]: It is dangerous to block fresh IPs used for phishing. Hoster may ban them and sell ip to legal client.

**Pull request closed without merging**